src/Features/Core/Portable/MakeMethodAsynchronous/AbstractMakeMethodAsynchronousCodeFixProvider.cs

```diff
@@ -103,7 +98,7 @@ internal abstract class AbstractMakeMethodAsynchronousCodeFixProvider : CodeFixP
             if (isOrdinaryOrLocalFunction && !methodSymbolOpt.Name.EndsWith(AsyncSuffix))
             {
                 return await RenameThenAddAsyncTokenAsync(
-                    keepVoid, document, node, methodSymbolOpt, cancellationToken).ConfigureAwait(false);
+                    keepVoid, isEntryPoint, document, node, methodSymbolOpt, cancellationToken).ConfigureAwait(false);
```

Comment:
instead of passing this in here, wouldn't you just want to not even do the rename if this is the entrypoint?  i.e. hoist ```!isEntryPoint``` into the if-check.